dev/io.openliberty.data.internal.persistence/src/io/openliberty/data/internal/persistence/cdi/FutureEMBuilder.java

```diff
@@ -169,6 +174,8 @@ EntityManagerBuilder createEMBuilder() {
                         resourceName = javaCompName;
                     }
                 } catch (NamingException x) {
+                    if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled())
+                        Tr.debug(this, tc, "Ignoring lookup failure for " + javaCompName + " : " + x);
```

Comment:
To avoid filling the logs with stacks of expected exceptions,
```suggestion
                        Tr.debug(this, tc, javaCompName + " is not available in JNDI");
```